lms/djangoapps/commerce/http.py

```diff
@@ -0,0 +1,22 @@
+""" HTTP-related entities. """
+
+from django.utils.translation import ugettext as _
+from rest_framework.status import HTTP_503_SERVICE_UNAVAILABLE, HTTP_200_OK
+
+from util.json_request import JsonResponse
+
+
+class DetailResponse(JsonResponse):
+    """ JSON response that simply contains a detail field. """
+
+    def __init__(self, message, status=HTTP_200_OK):
+        data = {'detail': message}
+        super(DetailResponse, self).__init__(object=data, status=status)
+
+
+class ApiErrorResponse(DetailResponse):
+    """ Response returned when calls to the E-Commerce API fail or the returned data is invalid. """
+
+    def __init__(self):
+        message = _('Call to E-Commerce API failed. Order creation failed.')
```

Comment:
Since this isn't intended to be a public API, do you think it's necessary to mark this message for translation?